Sources/PipecatClientIOS/RTVIError.swift

```diff
@@ -114,7 +114,7 @@ public struct AsyncExecutionError: RTVIError {
 public struct OtherError: RTVIError {
     public let message: String
     public let underlyingError: Error?
-    
+
     public init(message: String, underlyingError: Error? = nil) {
         self.message = message
         self.underlyingError = underlyingError
```

Comment:
not sure if it makes sense in iOS, but I added an `UnsupportedFeatureError` for when features are called that maybe only some transports support, like screensharing.

Reply:
Done.